soroban-sdk/src/testutils.rs

```diff
@@ -191,3 +157,32 @@ pub(crate) fn random<const N: usize>() -> [u8; N] {
     rand::thread_rng().fill_bytes(&mut arr);
     arr
 }
+
+#[cfg(not(target_family = "wasm"))]
+use super::TryFromVal;
+#[cfg(not(target_family = "wasm"))]
+use crate::env::xdr::{Hash, ScAddress, ScObject, ScVal};
+#[cfg(not(target_family = "wasm"))]
+use crate::Address;
+#[cfg(not(target_family = "wasm"))]
+impl Address {
```

Comment:
I think I was misunderstood. For consistency with the other types the impl should live with the type, and there should be a trait that lives here with the same name that is being implemented. Have a look at the traits above and their respective implementations for how they work with their type.

Reply:
Oh, sorry, I can't read. Updated for consistency.

FWIW this reminded me that the current approach is pretty IDE-unfriendly. Not only you won't get these functions in auto-completion by default, but also rust analyzer fails to properly import it (it doesn't insert `as _` automatically). So we actually may think of revisiting this (not in this PR obviously).

Reply:
> rust analyzer fails to properly import it (it doesn't insert `as _` automatically

+1 We could rename the traits to AddressTestutils.